en/faq/nginx-proxy.md

```diff
@@ -30,7 +30,91 @@ server {
         proxy_set_header X-Forwarded-Proto  $scheme;
         proxy_read_timeout          1m;
         proxy_connect_timeout       1m;
-        proxy_pass                          http://127.0.0.1:3000; # l'URL de l'instance Node.js ici
+        proxy_pass                          http://127.0.0.1:3000; # mettre l'URL de l'instance Node.js ici
+    }
+}
+```
+
+# Utiliser nginx avec des pages générées et un proxy de cache en solution de secours
+
+Si vous avez un site web très important avec du contenu changeant régulièrement, vous voudriez bénéficier des capacité de génération de Nuxt et de [nginx caching](https://www.nginx.com/blog/nginx-caching-guide).
+
+Vous trouverez ci-dessous un exemple de configuration. Gardez à l'esprit que :
+- le répertoire cracine doit être le même que celui définit par [la configuration du répertoire de génération](/api/configuration-generate#dir)
```

Comment:
cracine?

Reply:
"racine" ^_^'

"- root folder should be the same as set by (...)"

Reply:
corrigé dans le commit https://github.com/vuejs-fr/nuxt/pull/42/commits/6e936f2aff30512447bbda7cb211df52b8eed7a0